src/client/pages/Landing/landingPageData.ts

```diff
@@ -1,19 +1,48 @@
-import { LocalePath, Market } from 'components/utils/CurrentLocale'
+import { Market } from 'components/utils/CurrentLocale'
 import { checkFeature } from 'utils/checkFeature'
 import { Feature } from 'shared/clientConfig'
+import { AlternateLinksData, CanonicalLinksPerLocale } from 'src/routes'
 
-type AlternateLinksData = {
-  hrefLang: string
-  locale: LocalePath
-}[]
+export const LandingPageCanonicalLinks: CanonicalLinksPerLocale = {
+  se: 'https://www.hedvig.com/se/new-member',
+  'se-en': 'https://www.hedvig.com/en-se/new-member',
+  no: 'https://www.hedvig.com/no/new-member',
+  'no-en': 'https://www.hedvig.com/no-en/new-member',
+  dk: 'https://www.hedvig.com/dk/new-member',
+  'dk-en': 'https://www.hedvig.com/se/new-member',
```

Comment:
nit: Maybe we could just create these urls dynamically, it's easy to mistype :D 
```suggestion
  'dk-en': 'https://www.hedvig.com/dk-en/new-member',
```

Reply:
Yeah, apparently it is :D I'll look into it, thanks!

Reply:
Generated the canonical links now, good suggestion 👍